File: Python/array/design_parking_system.py
class ParkingSystem:
    def __init__(self, big: int, medium: int, small: int):
        self.remain = [big, medium, small]

    def addCar(self, carType: int) -> bool:
        if self.remain[carType - 1] > 0:
            self.remain[carType - 1] -= 1
            return True
        else:
            return False

File: core/tmp/Java/src/com/dbc/algorithm/Array/DesignParkingSystem.java
package com.dbc.algorithm.Array;

public class DesignParkingSystem {
    private int[] remain;
    public DesignParkingSystem(int big, int medium, int small) {
        remain = new int[]{big, medium, small};
    }

    public boolean addCar(int carType) {
        if (remain[carType - 1] > 0) {
            remain[carType - 1] -= 1;
            return true;
        } else {
            return false;
        }
    }
}
